client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -56,7 +105,13 @@ interface ConnectedOrganization {
   handle: ConnectionHandle;
 }
 
+const userInfo: Ref<ClientInfo | null> = ref(null);
```

Comment:
This variable is unused. I don't think it's necessary to leave the commented bits in the `<template>` and `<script>` sections, an issue should be enough.